recipe/meta.yaml

```diff
@@ -26,17 +26,24 @@ requirements:
     - toolchain
     - cmake
     - eigen
-    - numpy x.x
+    - numpy 1.7.*   # [py27]
+    - numpy 1.9.*   # [py35]
```

Comment:
I added numpy  1.8 with openblas for all pythons in case you want to simplify this. (As far as np1.7 we can do only py2.7 at the moment.)